### DISCUSSION TITLE
Time for initialization

### DISCUSSION THREAD

**Discussion by Robertarduino** [ClassicDIY/SkyeTracker#39] [Q&A] [answered]
How long take initialization sequence? In my system after reaching max East, system wait 4min.

**Comment by graham22** [comment, marked as answer]
It depends on the actuator length an speed that is configured using the app, (line 52 in LinearActuatorNoPot.cpp)
My actuators are 3.5 mm/sec @ 300mm length so it initializes in about 100 seconds.
